mgmt_cli_test.py

```diff
@@ -509,7 +511,9 @@ def test_backup_multiple_ks_tables(self):
         self.log.debug('tables list = {}'.format(tables))
         # TODO: insert data to those tables
         backup_task = mgr_cluster.create_backup_task(location_list=self.locations)
-        backup_task.wait_for_status(list_status=[TaskStatus.DONE], timeout=10800)
+        backup_task_status = backup_task.wait_and_get_final_status(timeout=1500)
```

Comment:
just a question here... the previous timeout was `10800` and now it is `1500`... will it be enough?

Reply:
Yes. The whole test doesn't even take this long. If the task didn't end until this point, I want it to time out.